openvdb/openvdb/tree/InternalNode.h

```diff
@@ -1029,6 +1030,18 @@ InternalNode<ChildT, Log2Dim>::nodeCount(std::vector<Index32> &vec) const
     vec[ChildNodeType::LEVEL] += count;
 }
 
+template<typename ChildT, Index Log2Dim>
+inline void
+InternalNode<ChildT, Log2Dim>::activeTileNodeCount(std::vector<Index32>& vec) const
+{
+    assert(vec.size() > ChildNodeType::LEVEL);
+    const auto countTile = mValueMask.countOn();
+    const auto countChild = mChildMask.countOn();
```

Comment:
why compute countCount? If there are no child nodes "for (auto iter = this->cbeginChildOn(); iter; ++iter)" will terminate as fast as a call to  mChildMask.countOn() (because the iterator works on 64 words) and if there are any child nodes you just wasted time computing  countChild. If you really insist on checking the child bit mask (whish I would advise against since the iterator is super fast) then instead call mChildMask.isOff() which might perform an early-out